src/core/scheduler/scheduler.cc

```diff
@@ -325,9 +337,10 @@ void Graph::RunGraph() {
     }
   }
 
+  EvaluateTimeElapsed();
+
   // increment iteration counter
   step();
```

Comment:
Remember to clear the elapsed time and the iteration when resetting the graph.

Reply:
Addressed

Added iteration_=0 in Graph::reset, the elapsed time will be cleared in Graph::rest together with node as well